core/src/main/java/org/pentaho/reporting/platform/plugin/PentahoReportEnvironment.java

```diff
@@ -175,7 +185,13 @@ public String getEnvironmentProperty( final String key ) {
       }
 
       if ( key.startsWith( "session:" ) ) { //$NON-NLS-1$
-        final Object attribute = session.getAttribute( key.substring( "session:".length() ) ); //$NON-NLS-1$
+        String sessionKey = key.substring( "session:".length() );
```

Comment:
You need a unit test for this new functionality in `PentahoReportEnvironment#getEnvironmentProperty( String ) ` for` IPentahoSession.TENANT_ID_KEY`

Please add to existing test class https://github.com/pentaho/pentaho-platform-plugin-reporting/blob/master/core/src/test/java/org/pentaho/reporting/platform/plugin/PentahoReportEnvironmentTest.java

Reply:
@dcleao , While writing test cases I found this block is already inside `if ( session != null )` condition and the `getSessionTenantId()` will never go inside `if ( session == null )` condition.

Should we move the logic out of the `if ( session != null )` or remove `if ( session == null )` condition.

Reply:
Thanks @dcleao , Will make changes according to that.